packages/expo-document-picker/src/ExpoDocumentPicker.web.ts

```diff
@@ -73,9 +73,10 @@ function readFileAsync(targetFile: File, base64: boolean = true): Promise<Docume
       reject(new Error(`Failed to read the selected media because the operation failed.`));
     };
     reader.onload = ({ target }) => {
-      const uri = (target as any).result;
+      const base64 = (target as any).result;
       resolve({
-        uri,
+        uri: URL.createObjectURL(targetFile),
+        base64,
```

Comment:
Returning base64 as an “extra” field instead of using it as the uri. Personally, I’m even in favor of removing it altogether. Since we already return the File object, users can easily build a FileReader themselves. (We could export a small utility for that too, but maybe that’s overkill.)